packages/ubuntu_wsl_setup/lib/main_win.dart

```diff
@@ -0,0 +1,97 @@
+import 'dart:io';
+
+import 'package:flutter/widgets.dart';
+import 'package:subiquity_client/subiquity_client.dart';
+import 'package:subiquity_client/subiquity_server.dart';
+import 'package:ubuntu_service/ubuntu_service.dart';
+import 'package:ubuntu_wizard/app.dart';
+import 'package:ubuntu_wizard/utils.dart';
+import 'package:ubuntu_wsl_setup/services/tcp_socket.dart';
+
+import 'app.dart';
+import 'args_common.dart';
+import 'installing_state.dart';
+import 'routes.dart';
+import 'services/journal.dart';
+
+Future<void> main(List<String> args) async {
+  final options = parseCommandLine(args, onPopulateOptions: (parser) {
+    addCommonCliOptions(parser);
+    parser.addOption(
+      'distro-name',
+      valueHelp: 'WSL Distribution name',
+      help: 'The name under which the distribution is being registered on WSL.',
+      defaultsTo: const String.fromEnvironment(
```

Comment:
is it intentionally a Dart define and not using Platform.environment?

Reply:
Yes, it's intentional. That piece of information is known at compile time. I want to have that value defined by the build system instead of the execution runtime to reduce the chance of mistakes due user defining a variable with that name and that influencing the behavior of the OOBE. It might be a theoretical risk at this point, but for now this seems a stronger approach.